aws-auth-cognito/src/test/java/com/amplifyframework/auth/cognito/helpers/SessionHelperTests.kt

```diff
@@ -34,6 +40,14 @@ class SessionHelperTests {
         expiration = 0
     )
 
+    // Encoded Date Time: April 30, 2024 02:00AM
+    private val currentInstant = Instant.ofEpochSecond(1714467600)
+
+    @After
+    fun teardown() {
+        unmockkStatic(Instant::class)
```

Comment:
Do you feel that it was better to mock Instant.now() static method versus using a real implementation? I think your test would work just the same without it since you aren't asserting on actual expiration values but just that the credential is valid or not.

Reply:
That's a good point! I think when I first started writing these, I had a more complicated test setup and when I made them simpler, I just left the mocking. I've removed the mocking and the tests work just fine still 👌